tests/Yaapii.Xml.Xambly.Tests/Directive/DirectivesTests.cs

```diff
@@ -51,6 +83,21 @@ public void ParsesIncomingGrammar()
                 new Yaapii.Atoms.Enumerable.LengthOf(dirs).Value() == 1);
         }
 
+        /// <summary>
+        /// Directives takes Xambly grammar as IText
+        /// </summary>
+        [Fact]
+        public void ParsesIncomingGrammarAsIText()
```

Comment:
I think the method name should be `ParsesIncomingGramerFromIText` because the grammer is parsed from a class implementing IText.

Reply:
Oh yes

Reply:
@koeeenig changed